.github/workflows/codeql.yml

```diff
@@ -3,8 +3,10 @@ name: "Code Scanning - Action"
 on:
   push:
     branches: [master]
+    branches: [dynamic_programming_knapsack]
```

Comment:
That may not work. You might want to run on all branches instead.
If so, please create another issue to discuss. 🙂